static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -296,11 +301,47 @@ type SpecialFields = {
   'user.display': SpecialField;
 };
 
+const DownloadCount = styled('span')`
+  padding-left: 6px;
+`;
+
 /**
  * "Special fields" either do not map 1:1 to an single column in the event database,
  * or they require custom UI formatting that can't be handled by the datatype formatters.
  */
 const SPECIAL_FIELDS: SpecialFields = {
+  // This is a custom renderer for a field outside discover
+  // TODO - refactor code and remove from this file or add ability to query for attachments in
+  attachments: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
+
+      const items: MenuItemProps[] = attachments.map(attachment => ({
+        key: attachment.id,
+        label: attachment.name,
+        onAction: () => window.open(attachment.url), // TODO - validate if this works
```

Comment:
```suggestion
        onAction: () => window.open(attachment.url),
```

Seems like it works, right?

Reply:
I haven't got attachement download to work at all in localhost, only in production, not sure why. I'd merge want to merge this PR and test it deployed.

Reply:
Validated, it works!